ISS/views/forum.py

```diff
@@ -887,6 +891,9 @@ def POST(self, request, poster_id):
 
             threads.update(forum=form.cleaned_data['target_forum'])
 
+            for vote in votes:
+                vote.delete()
```

Comment:
Querysets have a `delete` method too, which delete all the members of the QS in one query, so you can simply do `votes.delete()` here